### ISSUE TITLE
Equestripedia redirects don't go to the article, they only go to the main page

### ISSUE THREAD

**Issue by Forgilageord** [KevinPayravi/indie-wiki-buddy#463]
Tried with multiple articles, all of which do exist on Equestripedia, but while indie wiki buddy performs the redirect from mlp.fandom.com, it only ever goes to the Equestripedia homepage.

**Comment by KevinPayravi** [comment]
Good catch, thanks. This is similar to issue https://github.com/KevinPayravi/indie-wiki-buddy/issues/432 for Miraheze wikis (their index.php entrypoint requires a `&title=Special%3ASearch` query param for the search to work, unlike most other MediaWiki wikis).

I've pushed up a fix, which will be available in the next update.

**Comment by SnorlaxMonster** [comment]
Specifically, it seems to be _because_ Equestripedia is a Miraheze wiki (they just happen to have a custom URL).

**Comment by KevinPayravi** [comment]
Fixed in `v3.5.0`, now available on both Chrome and Firefox.
